main/lsp/requests/completion.cc

```diff
@@ -244,6 +244,7 @@ unique_ptr<CompletionItem> getCompletionItemForKeyword(const LSPConfiguration &c
 unique_ptr<CompletionItem> LSPLoop::getCompletionItemForSymbol(const core::GlobalState &gs, core::SymbolRef what,
                                                                core::TypePtr receiverType,
                                                                const core::TypeConstraint *constraint,
+                                                               const core::Loc queryLoc, string_view prefix,
```

Comment:
Do you need the prefix string here, or would its length be sufficient?

Reply:
The length would be sufficient, but I kind of like the `prefix` as a public API. I could go either way.